config/locales/controllers/en.yml

```
@@ -101,6 +101,9 @@ en:
       page_title: Terms of Service
     tos_faq:
       page_title: Terms of Service FAQ
+  invite_requests:
+    index:
+      browser_page_title: Invitation Requests | Archive Of Our Own
```

Following the format we've started (alongside the above change)
```suggestion
      page_title: Invitation Requests
```